lib/client/session.go

```diff
@@ -78,6 +79,10 @@ type NodeSession struct {
 
 	terminal *terminal.Terminal
 
+	// forceDisconnect if we should immediately disconnect upon finish instead of waiting for the remote status.
+	// This value must always be accessed atomically.
+	forceDisconnect int32
```

Comment:
Is this properly aligned for atomics?

Reply:
Yep. 32-bit atomics are not required to be 64-bit aligned on platforms like ARMv7. Said misalignment bug only happens with 64-bit atomics. See the footnote for the Go atomic package.